### DISCUSSION TITLE
how do you add a logo and description to a topic?

### DISCUSSION THREAD

**Discussion by spdegabrielle** [community/community#22344] [New to GitHub]
<p>Lots of the topics lack the relevant logo adn description.</p>
<p>how do you add a logo and description to a topic? (like <a href="https://github.com/topics/clojure" rel="nofollow noopener">https://github.com/topics/clojure</a> )</p>
<p>I’d like to add the Racket logo (<a href="https://github.com/racket" rel="nofollow noopener">https://github.com/racket</a>) to the Racket topic: <a href="https://github.com/topics/racket" rel="nofollow noopener">https://github.com/topics/racket</a></p>
<p>Kind regards,</p>
<p>Stephen</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi Stephen!</p>
<p>community-contributed content for topics lives in <a href="https://github.com/github/explore" rel="nofollow noopener">https://github.com/github/explore</a> - you can add more things there.</p>
<p>Cheers,</p>
<p>Dennis</p>
